studies/india_districts/india_state_level.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+from typing import Dict, Optional, Sequence
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+
+from adaptive.plots import plot_RR_est
+from adaptive.estimators import gamma_prior
+from adaptive.smoothing import convolution
+from adaptive.utils import cwd, days
+from adaptive.etl.covid19india import download_data, load_statewise_data, get_time_series
+from statsmodels.regression.linear_model import OLS
+from statsmodels.tools import add_constant
+
+def project(dates, R_values, smoothing, period = 7*days):
+    julian_dates = [_.to_julian_date() for _ in dates[-smoothing//2:None]]
+    return OLS(
+        RR_pred[-smoothing//2:None], 
+        add_constant(julian_dates)
+    )\
+    .fit()\
+    .predict([1, julian_dates[-1] + period])[0]
+
+if __name__ == "__main__":
+    root = cwd()
+    data = root/"data"
+
+    # model details 
+    gamma      = 0.2
+    smoothing = 15
+    CI        = 0.95
+
+    download_data(data, 'state_wise_daily.csv')
+
+    state_df = load_statewise_data(data/"state_wise_daily.csv", data/"india_state_code_lookup.csv")
```

Comment:
can we create a dictionary in python code for the state code lookups? It can live in the `adaptive.etl.covid19india` package